### PR TITLE
fix spelling error and remove redundant code

### PR DESCRIPTION
**Pull request by xujingjing-cmss**
Signed-off-by: xujingjing xujingjing_yewu@cmss.chinamobile.com

What type of PR is this?
/kind cleanup

What this PR does / why we need it:
fix spelling error and remove redundant code.

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

### CODE REVIEW

File: edge/pkg/common/util/config.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func GenerateTestCertificate(path string, certFileName string, keyFileName strin
 		SerialNumber:          big.NewInt(1234),
 		Subject: pkix.Name{
 			Country:      []string{"test"},
-			Organization: []string{"testor"},
+			Organization: []string{"tester"},
```
Comment (zc2638) [inline edge/pkg/common/util/config.go:44]: It seems that the two words mean the same thing.

Comment (xujingjing-cmss) [reply edge/pkg/common/util/config.go:44]: It seems that there isn't such a word 0.0

Comment (fisherxu) [reply edge/pkg/common/util/config.go:44]: ;-)

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**